frame/collective/src/benchmarking.rs

```diff
@@ -0,0 +1,162 @@
+// Copyright 2020 Parity Technologies (UK) Ltd.
+// This file is part of Substrate.
+
+// Substrate is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+
+// Substrate is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+
+// You should have received a copy of the GNU General Public License
+// along with Substrate.  If not, see <http://www.gnu.org/licenses/>.
+
+//! Staking pallet benchmarking.
+
+use super::*;
+
+use frame_system::RawOrigin as SystemOrigin;
+use frame_benchmarking::{benchmarks_instance, account};
+
+use frame_system::Module as System;
+use crate::Module as Collective;
+
+const SEED: u32 = 0;
+
+benchmarks_instance! {
+	_{
+		// User account seed
+		let u in 1 .. 1000 => ();
+		let m in 1 .. 1000 => ();
+	}
+
+	set_members {
```

Comment:
You should also have a bunch of "old members" in storage who are different than these members.

Reply:
Old members added.